geoips/plugins/yaml/product_defaults/tpw/TPW-CIMSS.yaml

```diff
@@ -15,8 +15,16 @@ spec:
         inverse: null
   colormapper:
     plugin:
-      name: tpw_cimss
-      arguments: {}
+      name: matplotlib_linear_norm
+      arguments:
+        cmap_name: tpw_cimss
+        cmap_source: ascii
```

Comment:
@evrose54 Note ascii palettes require passing cmap_source: ascii.

cmap_source is one of
* matplotlib (built in matplotlib colormaps)
* ascii (ascii palette on disk, installed or full path)
* geoips (pulls from a geoips colormapper plugin)